client/src/components/content/RobotContainer/RobotContainer.jsx

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-alert */
+import React from 'react';
+import { Col, Row, Typography } from 'antd';
+import { PlayCircleOutlined, EditOutlined } from '@ant-design/icons';
+import PropTypes from 'prop-types'
+import styles from './RobotContainer.module.css';
+
+const { Title } = Typography;
+
+/**
+ * @component
+ * @description Provides a prototype-box for all robots to be displayed in the Robot Overview 
+ * @category Client
+ */
+const RobotContainer = (props) => {
```

Comment:
Stylewise you might consider using border radius to round the corners of all the container. It`s not a must just a nice to have ;)

Reply:
✅